lib/src/sync/common.dart

```diff
@@ -16,6 +16,7 @@ import 'web_driver.dart';
 import 'web_element.dart';
 
 const String elementStr = 'ELEMENT';
+const String w3cElementStr = 'element-6066-11e4-a52e-4f735466cecf';
```

Comment:
This is the "web element constant", reference here: https://www.w3.org/TR/webdriver/#elements

It replaces the use of 'ELEMENT' in the W3C spec. 

I am curious how one arrives at *this* as a magic constant in a spec, but not enough to hunt down the history here.